src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Main {
+    static final int ARRAY_LIMIT = 10;
```

Comment:
what about access modifier?

Reply:
I have changed access modifiers of constants in Main and FigureSupplier classes.